### DISCUSSION TITLE
Mouse injector not working

### DISCUSSION THREAD

**Discussion by RADIOACTIVEshrek** [garungorp/MouseInjectorDolphinDuck#10] [General]
Heya. Been trying to play multiple different games with the mouse injector but it hasn't been detecting any of the games. This is after I got my new harddrive.
 It has been rightfully detecting the emulator, but none of the games. I've tested Geist, Cold Winter, Dirge of Cerberus, Die Hard Vendetta and Black. All used to work before my new harddrive, but after, none of them have worked anymore.

**Comment by YourKiss** [comment]
Try to delete all States of game and restard emulator with injector, and don't use Load\Save State feature

**Reply by RADIOACTIVEshrek** [reply]
Thanks for the help, that worked.
